scanners/subfinder/examples/securecodebox.io/active/findings.yaml.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: the secureCodeBox authors
```

Comment:
this license file probably doesn't need to exist?
the file already has a license header